### PR TITLE
chore: extract yaml file for schema review configuration

### PR DESCRIPTION
**Pull request by ecmadao**
- Using yaml file for schema review configuration.
- Update the localization.
图片
图片

### CODE REVIEW

File: scripts/get_schema_review_rules.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+# TODO: use version to fetch the lastest file
+FILE="sqlReviewConfig.yaml"
+URL="https://raw.githubusercontent.com/bytebase/bytebase/main/frontend/src/types/$FILE"
```
Comment (ecmadao) [inline scripts/get_schema_review_rules.sh:4]: As the console hasn't published the YAML file into a specific version, we have to use the URL on the main branch.

File: locales/zh.json [hunk 2]
```diff
@@ -323,7 +323,137 @@
...
+      "mysql-prod": {
+        "env": "Prod",
+        "title": "MySQL 审查规则"
+      },
+      "mysql-dev": {
+        "env": "Dev",
+        "title": "MySQL 审查规则"
+      }
+    },
+    "rule": {
```
Comment (tianzhou) [inline locales/zh.json:359]: out of the scope of this PR. ideally, we can also pull the localization file from the console.

**Review by tianzhou — APPROVED** [review]
LGTM

**Review by boojack — APPROVED** [review]
LGTM overall

**Pull request merged**